README.md

```diff
@@ -256,9 +256,28 @@ ARM_TENANT_ID=<Tenant ID>
 ARM_CLIENT_ID=<Service principle ID>
 
 ````powershell
-az login
+az login --use-device-code
 
 terarform apply
 # on prompt if successfull type 'yes'
 
-````
\ No newline at end of file
+````
+
+To run in Kubinates
+
+prerquisits: please install the following (docker you should have already if following on from above)
+Kubectl -  https://kubernetes.io/docs/tasks/tools/
+minikube - https://minikube.sigs.k8s.io/docs/start/
+
+then run the following:
+````powershell
+docker build --target production --tag todo-app1:prod .
```

Comment:
You've switched to `todo-app1:prod` here but your deployment.yaml still says `todo-app:prod`, either is fine but they should match! (and anywhere else, e.g. below where we load the image)

Reply:
thanks 
yes i added the 1 to overcome the image upload was the right one at the time
i will correct

Reply:
changed todo-app1:prod to todo-app:prod in both places